cmd/chaos-controller-manager/provider/suit_test.go

```diff
@@ -58,7 +58,7 @@ func TestProvider(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
-	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
+	logf.SetLogger(log.NewZapLoggerWithWriter(ginkgo.GinkgoWriter))
```

Comment:
```suggestion
	logf.SetLogger(log.NewZapLoggerWithWriter(GinkgoWriter))
```

Reply:
This should be done only in this file or every file @STRRL ?

Reply:
for every file which uses dot imports with "github.com/onsi/ginkgo"